src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -446,7 +474,9 @@ inlineToOpenDocument o ils
      | writerWrapText o == WrapPreserve
                   -> return $ preformatted "\n"
      | otherwise  ->return space
-    Span attr xs  -> withLangFromAttr attr (inlinesToOpenDocument o xs)
+    il@(Span attr xs)-> do
+                       S.when (isDisplayMath il) (setDisplayMath True)
```

Comment:
This seems to set `stDisplayMath` to True at the first occurrence of display math; it then stays True throughout the rest of the document -- at least I don't see anywhere where it gets reset. Is this the intent?  AS noted above, I'm confused about what work this is doing.

Reply:
The idea was to put display math into an own paragraph and center it. I have changed the implementation so that it is not needed anymore. Now I set the `anchor-type` of the formula to `paragraph`. which is what pandoc did before. Now I put the properties like `style:vertical-pos` and `style:vertical-rel` into a style, because these properties are not supported directly on a `draw:frame`. The document produced by pandoc was not compliant to the spec, I think.